### ISSUE TITLE
Experimentor's temporal starlane crosses over system

### ISSUE THREAD

**Issue by Oberlus** [freeorion/freeorion#1559]
### Environment

* **FreeOrion Version**: 2017-05-02.ded1942_Test_Win32
* **Operating System**: Windows 7
* **Graphic card used**: Nvidia 660 GTX
* **Fetched as** 
  * Binary release
  * Weekly development build

### Description

The Experimentors created a (temporal) starlane that crosses over one planetary system.
It does not cross exactly over another starlane, just over a system, but can be considered as crossing over starlanes since that system has starlanes at both sides of the system.
![starlane_crossing_over_system](https://cloud.githubusercontent.com/assets/28650914/25998139/d6ec4e40-371f-11e7-85c3-9e30879f4a62.jpg)

### Expected Result

Starlane should not cross over systems. It could have been attached to that system but not to another behind it, as is the case.

### Steps to reproduce

Save game attached.
[laenfa_experimentors_269.sav.zip](https://github.com/freeorion/freeorion/files/996107/laenfa_experimentors_269.sav.zip)

**Comment by Oberlus** [comment]
This bug is still alive.

![ExperimentorCrossingStarlanes](https://user-images.githubusercontent.com/28650914/87779997-12578d80-c82e-11ea-9d99-3ff0ceb8e672.png)

**Comment by geoffthemedio** [comment]
Was this fixed by dfd2c6ce2363e4738d6e9b2571b3307e8d9bdd64 ?

**Comment by Oberlus** [comment]
No. Current MP game (19th) had this issue, and we are playing version 580d50d, january 2022, and dfd2c6c is december 2021.
